whois-commons/src/main/java/net/ripe/db/whois/common/apiKey/ApiKeyUtils.java

```diff
@@ -84,7 +91,7 @@ public static boolean isAPIKeyRequest(final String authHeader) {
     }
 
     public static String getApiKeyId(final String authHeader) {
-        if(authHeader == null || !authHeader.startsWith("Basic ")) {
+        if (authHeader == null || !authHeader.startsWith("Basic ")) {
```

Comment:
May be "Basic ".startsWith(authHeader) , it will do null check as well

Reply:
Done